test/cases_filter_timestamp/2.callbacks.c

```diff
@@ -0,0 +1,16 @@
+#include <metababel/metababel.h>
```

Comment:
We can merge sink and filter in the same test.  Like i did for array.
Then we can meta-generate this kind of tests